scss/custom/_modal.scss

```diff
@@ -22,8 +22,8 @@
   .modal-header,
   .modal-footer {
     // Fixed for older phones
-    display: flex;
     z-index: 1;
+    display: flex;
```

Comment:
@lightblader Could you explain why this change is required?

Reply:
Because due to linter errors the version bump command cannot be executed